### ISSUE TITLE
Don't crash when a scene isn't defined in entities.json

### ISSUE THREAD

**Issue by ericlathrop** [SplatJS/splat-ecs#31]

**Comment by CaldwellYSR** [comment]
Can you give more info? It doesn't crash for me when I don't have a scene in entities unless I try to switch to that scene... What are you wanting it to do when you try to switch to a scene with no entities?

**Comment by ericlathrop** [comment]
Yeah, it shouldn't crash when you switch to it. It should just pretend there's no entities.

**Comment by CaldwellYSR** [comment]
[This](https://github.com/CaldwellYSR/splat-ecs/blob/no_entity_scene/lib/game.js#L75) partially works... I was able to successfully load a test scene as "first" and then in the onEnter script switch scenes back to the main scene without it crashing. When trying to load the empty scene it just sits with the loading bar full and no errors in the console.
